docs/source/guide/lre-3-options.md

```diff
@@ -0,0 +1,282 @@
+---
+jupytext:
+ text_representation:
+   extension: .md
+   format_name: myst
+   format_version: 0.13
+   jupytext_version: 1.11.1
+kernelspec:
+ display_name: Python 3
+ language: python
+ name: python3
+---
+
+
+# What additional options are available when using LRE?
+
+
+In [](lre-1-intro.md), {func}`.execute_with_lre` was used to calculated the error mitigated expectation values
+In this section, we will outline the optional arguments that can be used and adjusted with this technique.
+
+
+```python
+lre_value = mitiq.lre.execute_with_lre(
+    circuit,
+    executor,
+    degree,
+    fold_multiplier,
+    folding_method=<"noise scaling method imported from zne.scaling.folding">,
+    num_chunks=<"number of chunks to group a large circuit into">,
+)
+```
+
+The hyperparameters that can be controlled are:
+
+
+- `degree`: to modify the extrapolating polynomial
+- `fold_multiplier`: to control how the circuit is scaled
+- `folding_method`: to choose the unitary folding method
+- `num_chunks`: to alter the sampling cost
+
+
+## Controlling how the noise-scaled circuits are created
+
+
+### Extrapolating polynomial
+
+
+Depending on the choice of the degree of the polynomial, how each noise-scaled circuit is scaled varies. This
+in turn influences how the sample matrix for multivariate Richardson extrapolation is created.
+
+
+Suppose we have an input circuit consisting of 4 layers.
+
+
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT, H
+
+
+q0, q1, q2, q3  = LineQubit.range(4)
+circuit = Circuit(
+   H(q0),
+   CNOT.on(q0, q1),
+   CZ.on(q1, q2),
+   CNOT.on(q2, q3),
+)
+
+
+print(circuit)
+```
+
+
+How the circuits are scaled is controlled by `degree`, `fold_multiplier`, and the number of layers in the input circuit. We choose `degree = 2` and `fold_multiplier = 2` for demonstration purposes.
+
+
+
+For `degree = 2`, the terms in the monomial basis are given below where the subscripts correspond to the 4 layers in the
+circuit.
+
+
+$$\{1, λ_1, λ_2, λ_3, λ_4, λ_1^2, λ_1 λ_2, λ_1 λ_3, λ_1 λ_4, λ_2^2, λ_2 λ_3, λ_2 λ_4, λ_3^2, λ_3 λ_4, λ_4^2\}$$
+
+
+```{seealso}
+These terms in the monomial basis are also used to define the rows of the sample matrix used for extrapolation.
+```
+
+
+```{code-cell} ipython3
+from mitiq.lre.multivariate_scaling import get_scale_factor_vectors
+
+scale_factors = get_scale_factor_vectors(circuit, degree=2, fold_multiplier=2)
+
+scale_factors
+```
+
+
+In the noise scaled circuits created using the above scale factor vectors:
```

Comment:
1. A comment as to where the numbers 5 and 9 come from in the above `scale_factors` variable is needed.
2. Correct me if I'm wrong, but at this point in the user guide, the reader might not know what relation the monomial basis has to scale factors. It's probably worth calling out that relationship. Otherwise it might seems confusing why you jumped straight from monomials to scale factors.